geopandas/tests/test_geoseries.py

```diff
@@ -328,6 +329,21 @@ def test_to_wkb(self):
     def test_to_wkt(self):
         assert_series_equal(pd.Series([self.t1.wkt, self.sq.wkt]), self.g1.to_wkt())
 
+    def test_repr_linearring(self):
```

Comment:
I wasn't sure where this test should live, there are some repr tests in test_pandas_methods but these seem to be about edge cases and not geometry printing specific. I've put this here near wkt/wkb tests. As an aside, the WKT was correctly showing LinearRing for these cases, was just an issue for the repr as it's roundtripped through wkb, presumably as that's more efficient.

Reply:
Looking at our existing tests, I moved it to test_pandas_methods.py. It's also not a super obvious place, but it fits better with the existing repr tests we already have there, I think. 

A difference with WKT conversion is that that is a vectorized method, while the repr formatting is done on scalar values (through the pandas extension interface, and thus living in the "pandas" tests, I assume), and so you have this additional conversion step from accessing those scalar values:

https://github.com/geopandas/geopandas/blob/c9c530d6c4e3f2aa44c96a699a931c2a3ead6058/geopandas/array.py#L1254-L1307